src/accelerate/accelerator.py

```diff
@@ -322,7 +323,7 @@ def __init__(
                 raise ImportError("Megatron is not installed. please build it from source.")
 
         if ipex_plugin is None:  # init from env variables
-            ipex_plugin = IntelPyTorchExtensionPlugin() if os.environ.get("IPEX_ENABLED", "false") == "true" else None
```

Comment:
Hello, removing this is leading to various test failures as we are trying to integrate accelerate in transformers trainer. We are planning to revert this line and wanted to check if there was a reason for removing this.